balancer/roundrobin/roundrobin_test.go

```diff
@@ -99,7 +100,11 @@ func TestOneBackend(t *testing.T) {
 	}
 	defer test.cleanup()
 
-	cc, err := grpc.Dial(r.Scheme()+":///test.server", grpc.WithInsecure(), grpc.WithBalancerBuilder(newBuilder()))
+	rr := balancer.Get("roundrobin")
```

Comment:
```go
package balancer

const RoundRobin = "roundrobin"
// or
var RoundRobinBuilder = ....
```

?

If not, I think a const in this test would be better than repeatedly using the same string.

Reply:
Added a const (a `var` actually) to roundrobin_test.

In theory, a user can re-register a roundrobin and `Get` will return the new one. So a exported variable in `balancer` package won't work.